draft-ietf-tls-esni.md

```diff
@@ -327,7 +327,7 @@ The client-facing server advertises a sequence of ECH configurations to clients,
 serialized as follows.
 
 ~~~~
-    ECHConfig ECHConfigList<4..2^16-1>;
+    ECHConfig ECHConfigList<1..2^16-1>;
```

Comment:
This is a game we play often.  There's a view that says that you don't set bounds like this.  Because then you get someone (like me) coming along to say that the minimum size of a single ECHConfig is 4 bytes.

Reply:
Fair enough :) Thanks for the context.